.evergreen/config.yml

```diff
@@ -3264,6 +3308,12 @@ buildvariants:
       batchtime: 20160 # Use a batchtime of 14 days as suggested by the CSFLE test README
     - testazurekms-fail-task
 
+- name: rhel8-test-lambda
+  display_name: AWS Lambda handler tests
+  run_on: rhel87-large
```

Comment:
Does this need to use rhel87-large? Or can we use rhel87-small instead?

Reply:
Small works